### PR TITLE
OBDAAC and SeaBASS Portal Addition

### PR DESCRIPTION
**Pull request by jacobtdawson**
# Overview

### What is the feature?

This PR adds two portals to EarthData-Search. One is for OB.DAAC and the other is for SeaBASS

### What is the Solution?

This simply follows the portal creation guide here: https://wiki.earthdata.nasa.gov/display/EDSC/Portal+Creation+Guide in order to add the two portals mentioned above.
I did this in a series of commits because of internal discussions about how our portal should look/work; please squash these, I don't want to mess up your commit history!

### What areas of the application does this impact?

The "portal" feature

# Testing

I have tested this locally on an instance running on our internal OBDAAC servers. It appears to run without issue, and the portals follow the format laid out by the portal creation guide and are similar to the already-present portals.

### Attachments

Here's an image of the two portals I've added:
image

# Checklist

- [NA] I have added automated tests that prove my fix is effective or that my feature works
- [NA] New and existing unit tests pass locally with my changes
- [x] I have performed a self-review of my own code
- [NA] I have commented my code, particularly in hard-to-understand areas
- [NA] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

File: portals/obdaac/config.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    "dataCenter": "NASA/GSFC/SED/ESD/GCDC/OB.DAAC"
+  },
+  "title": {
+    "primary": "OBDAAC",
+    "secondary": "Ocean Biology Distributed Active Archive Center"
+  },
+  "ui": {
+    "showNonEosdisCheckbox": true,
+    "showOnlyGranulesCheckbox": true,
+    "showTophat": false
```
Comment (eudoroolivares2016) [inline portals/obdaac/config.json:38]: We can remove this `"showTophat": false` key.

Comment (jacobtdawson) [reply portals/obdaac/config.json]: Sounds good, I assume that that defaults to false

File: portals/obdaac/config.json [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  },
+  "parentConfig": "edsc"
```
[the remark this answers is not in the document]
Comment (jacobtdawson) [reply portals/obdaac/config.json]: Sounds good! I've handled this and the other comment in both portals I'm adding

**Comment by eudoroolivares2016** [comment]
I think we'll just need to rebase this branch

**Comment by jacobtdawson** [comment]
> I think we'll just need to rebase this branch

Rebased, I didn't know that github had such a nice feature for that!

**Comment by eudoroolivares2016** [comment]
thank you! I can squash and merge when there is another approval from a maintainer

**Review by eudoroolivares2016 — APPROVED** [review]

**Review by trevorlang — APPROVED** [review]

**Pull request merged**